frontend/src/pages/signup/components/SignupForm/SignupForm.tsx

```diff
@@ -104,8 +131,17 @@ function SignupForm() {
   } = useVerificationCodeConfirm({
     verificationCode,
     verificationCodeErrorMessage,
+    successVerification,
   });
 
+  const getDisplayedVerificationErrorMessage = () => {
+    if (verificationStep === 'verified') {
+      return getFinalVerificationCodeErrorMessage();
+    } else {
+      return submitVerificationErrorMessage;
+    }
+  };
+
```

Comment:
p4: 이렇게 early return 해주면 컨벤션도 지킬 수 있고 가독성도 더 좋을 것 같은데 어때??

```suggestion
 const getDisplayedVerificationErrorMessage = () => {
  if (verificationStep === 'verified') {
    return getFinalVerificationCodeErrorMessage();
  }
  return submitVerificationErrorMessage;
};
```

Reply:
나도 범태 의견에 한 표 👍

Reply:
[refactor: early return 으로 변경](https://github.com/woowacourse-teams/2025-Fit-toring/pull/411/commits/0c19e546e204bb58621916a00196b7434a0b65b0)